src/nyc_trees/nyc_trees/settings/base.py

```diff
@@ -286,3 +286,5 @@
 # If geocoding a string produces no results, this string will be
 # appended for a second attempt.
 GEOCODE_FALLBACK_SUFFIX = ', New York, NY'
+
+TILER_URL = environ['TILER_HOST']
```

Comment:
I get the dislike for setting defaults in two different places, but in this case I think it is necessary because I view the Ansible world and the Django settings world as separate. Although unlikely, someone could deploy this app without our provisioning steps. In that case, they wouldn't get a default for this variable if they missed setting the environmental variable.